riddle.py

```diff
@@ -0,0 +1,18 @@
+'''Программа загадывает число от 0 до 1000. Необходимо угадать число за 10 попыток. Программа
+должна подсказывать «больше» или «меньше» после каждой попытки.'''
+
+from random import randint
+
+number = randint(1, 1000)
```

Comment:
Старайтесь не использовать магические числа. А ещё у вас неверно указан диапазон
```suggestion
number = randint(0, 1000)
```